modules/product/src/main/java/com/opengamma/strata/product/swap/type/OvernightRateSwapLegConvention.java

```diff
@@ -428,12 +428,12 @@ public StubConvention getStubConvention() {
    * the frequency to the start date, or subtracting it from the end date.
    * The roll convention provides the detailed rule to adjust the day-of-month or day-of-week.
    * <p>
-   * This will default to 'None' if not specified.
+   * This will default to 'EOM' if not specified.
    * 
    * @return the roll convention, not null
    */
   public RollConvention getRollConvention() {
-    return rollConvention != null ? rollConvention : RollConventions.NONE;
+    return rollConvention != null ? rollConvention : RollConventions.EOM;
```

Comment:
Why are we now defaulting to EOM? This is alot more restrictive in terms of which dates are acceptable?.

Reply:
EOM means "prefer EOM". So it can still roll on the 5th or the 20th, its just that if the roll starts on the 30th November it will be treated as EOM instead of 30th.